docs/src/content/core/typography.mdx

```diff
@@ -40,49 +43,51 @@ Then include it in your global stylesheet:
 @import '@pluralsight/ps-design-system-normalize';
 ```
 
-## CDN reference
+### CDN reference
 
 The normalize stylesheet should be used in most cases. But if you need a direct reference to the font file, it is located at:
 
 ```
 https://fonts.pluralsight.com/ps-tt-commons/v1/ps-tt-commons-variable-roman.woff2
 ```
 
-## Domain-restricted access
+### Domain-restricted access
 
 The CDN reference used by the Normalize stylesheet is domain-restricted. In other words, only requests coming from enumerated domains will succeed. If you have a new domain that you require new font access on, please contact #platform-ui in Slack.
 
-## Font weight
+## Type variables
+
+### Font weight
 
-10 font weights are available.
+It is recommended to use `psTypeFontWeightDefault` (400) & `psTypeFontWeightStrong` (600). Additionally there are 9 total weights available. Never use a font weight less than 400 on font sizes 18px and lower.
 
 <FontWeight />
 
-To apply the font weights, use the vars in css:
+Applying font weight using CSS vars:
```

Comment:
I like the terseness, but this isn't an English sentence. Seems like a departure from how we usually use complete sentences. 

What was your reason to change?

Reply:
Adjusted the wording preceding the example to:

Example of using CSS vars to adjust `{property}`: